### PR TITLE
no more redis in gettingstarted doc

### LINKED ISSUE #152: Update gettingstarted guide to remove redis requirement for logstash
With next beta redis is not required for logstash + redis might be deprecated. Update gettingstarted doc on logstash usage.

### PR DESCRIPTION
**Pull request by urso**
closes #152

### CODE REVIEW

File: docs/gettingstarted.asciidoc [hunk 1]
```diff
@@ -89,82 +89,79 @@ curl http://127.0.0.1:9200
...
 
 image:./images/packetbeat_logstash.png[Integration with Logstash]
 
-In this setup, the Beat shippers use the `RPUSH` Redis command to insert
-into a list stored in memory by Redis. Logstash reads from this key using the
-http://www.elastic.co/guide/en/logstash/current/plugins-inputs-redis.html[Redis
-input plugin] and then sends the transactions to Elasticsearch using the
+In this setup, the Beat shippers send events to Logstash. Logstash receives
+these events using the
+http://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html[Beats
```
Comment (urso) [inline docs/gettingstarted.asciidoc]: beats input plugin documentation currently missing

File: docs/gettingstarted.asciidoc [hunk 2]
```diff
@@ -89,82 +89,79 @@ curl http://127.0.0.1:9200
...
-same protocol as it is used between the
-https://github.com/elastic/logstash-forwarder[Logstash Forwarder] and Logstash.
+The simplest architecture for the Beat platform setup consists of the Beats
+shippers, Elasticsearch and Kibana. This is nice and easy to get started with
+and enough for networks with small traffic. It also uses the minimum amount of
+servers: a single machine running Elasticsearch + Kibana. The Beat shippers
+insert the transactions directly into the Elasticsearch instance.
+
+This article explains how to use the Beat together with Logstash to provide
+additional buffering and. Another important advantage is that you have the
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: buffering and ...?

File: docs/gettingstarted.asciidoc [hunk 3]
```diff
@@ -89,82 +89,79 @@ curl http://127.0.0.1:9200
...
-    # Set the host and port where to find Redis.
-    host: "127.0.0.1"
-    port: 6379
+    hosts: ["127.0.0.1:5044"]
 
-    # This option makes sense only for Packetbeat
-    save_topology: true
-------------------------------------------------------------------------------
+    # configure index prefix name
+    index: packetbeat
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: should we have packetbeat here? better beat or filebeat, as filebeat needs logstash

File: docs/gettingstarted.asciidoc [hunk 4]
```diff
@@ -89,82 +89,79 @@ curl http://127.0.0.1:9200
...
+      # disable tls for testing (TLS must be disabled in logstash too)
+      disabled: true
+------------------------------------------------------------------------------
 
-Then, configure Logstash to read from Redis and index into Elasticsearch. Here
-is a sample configuration that you can save under `/etc/logstash/conf.d/`:
+Next configure Logstash to listen on Port 5044 for incoming beats connections
+and index into Elasticsearch. The Beats platform sends the index and document
+type the Beat would use for indexing into Elasticsearch as additional meta data.
+Here is a sample configuration that you can save under `/etc/logstash/conf.d/`:
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: an example

File: docs/gettingstarted.asciidoc [hunk 5]
```diff
@@ -89,82 +89,79 @@ curl http://127.0.0.1:9200
...
+    tls:
+      # disable tls for testing (TLS must be disabled in logstash too)
+      disabled: true
+------------------------------------------------------------------------------
 
-Then, configure Logstash to read from Redis and index into Elasticsearch. Here
-is a sample configuration that you can save under `/etc/logstash/conf.d/`:
+Next configure Logstash to listen on Port 5044 for incoming beats connections
+and index into Elasticsearch. The Beats platform sends the index and document
+type the Beat would use for indexing into Elasticsearch as additional meta data.
```
Comment (monicasarbu) [inline docs/gettingstarted.asciidoc]: document type that the Beat ...

File: docs/gettingstarted.asciidoc (hunk 3 above)
Comment (urso) [inline docs/gettingstarted.asciidoc]: let me just remove this line (beat name is default)

**Comment by urso** [comment]
We require a new picture not including redis for this article.

**Comment by urso** [comment]
please notify me before merging so I can squash history.

File: docs/gettingstarted.asciidoc [hunk 6]
```diff
@@ -89,82 +89,77 @@ curl http://127.0.0.1:9200
...
-    host: "127.0.0.1"
-    port: 6379
+    hosts: ["127.0.0.1:5044"]
 
-    # This option makes sense only for Packetbeat
-    save_topology: true
-------------------------------------------------------------------------------
+    # configure logstash plugin to loadbalance events between
+    # configured logstash hosts
+    #loadbalance: true
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: Uncommenting a line should always have the default in. I suggest to make it false here.

File: docs/gettingstarted.asciidoc [hunk 7]
```diff
@@ -89,82 +89,77 @@ curl http://127.0.0.1:9200
...
 
-NOTE: If you want the Beat to monitor Redis traffic, it's better to change
-the port for the Redis instance that you use for collecting the traffic.
-Otherwise, you create a loop where the shipper captures the traffic it sends.
+    tls:
+      # disable tls for testing (TLS must be disabled in logstash too)
+      disabled: true
```
Comment (ruflin) [inline docs/gettingstarted.asciidoc]: As this is a default value, I suggest to comment it out (see loadbalance above)

**Comment by ruflin** [comment]
LGTM besides the two notes.

**Comment by tsg** [comment]
Should we add a note that you need to do `bin/plugin -i logstash-input-beats`?

**Comment by urso** [comment]
@tsg the plugin will be available by default with logstash 2 and maybe next logstash 1.5/1.6 release.

File: docs/gettingstarted.asciidoc (hunk 1 above)
Comment (tsg) [inline docs/gettingstarted.asciidoc]: we could link to the github repo in the meantime? That's what I did in the migration guide.

**Comment by tsg** [comment]
@urso ok, no need then.

File: docs/gettingstarted.asciidoc (hunk 1 above)
Comment (urso) [inline docs/gettingstarted.asciidoc]: sounds good

File: docs/gettingstarted.asciidoc (hunk 7 above)
Comment (urso) [inline docs/gettingstarted.asciidoc]: if section tls is not missing, disabled is false by default. It's kind of an option to comment out the section. Will remove 'disabled' as everyone seems to be confused by it.

**Comment by urso** [comment]
@ruflin @tsg update docs.

**Pull request merged**